gateway/ln-gateway/src/lib.rs

```diff
@@ -48,6 +48,8 @@ use crate::rpc::{
 
 const ROUTE_HINT_RETRIES: usize = 10;
 const ROUTE_HINT_RETRY_SLEEP: Duration = Duration::from_secs(2);
+/// LND HTLC interceptor can't handle SCID of 0, so start from 1
+const INITIAL_SCID: u64 = 1;
```

Comment:
This was a very important change. Without this, you can't receive into the federation via LND gateway because LND can't intercept HTLCs with an SCID of 0 (the previous value).